strings/strings_id.arb

```diff
@@ -83,13 +83,13 @@
   "repostedBy": "Diposting ulang oleh {actor}",
   "replyTo": "Membalas ke {actor}",
   "advancedButton": "Selanjutnya",
-  "notificationLikeReason": "{count, plural, other{Menyukai postingan Anda}}",
-  "notificationRepostReason": "{count, plural, other{Mem-posting ulang postingan Anda}}",
-  "notificationFollowReason": "{count, plural, other{Mengikuti Anda}}",
-  "notificationMentionReason": "{count, plural, other{Menyebut Anda}}",
-  "notificationQuoteReason": "{count, plural, other{Mengutip postingan Anda}}",
-  "notificationReplyReason": "{count, plural, other{Membalas postingan Anda}}",
-  "notificationTitleCounter": "{count, plural, other{{count} others}}",
+  "notificationLikeReason": "{count, plural, Lainnya{Menyukai postingan Anda}}",
+  "notificationRepostReason": "{count, plural, Lainnya{Posting ulang Postingan Anda}}",
+  "notificationFollowReason": "{count, plural, Lainnya{Mengikuti Anda}}",
+  "notificationMentionReason": "{count, plural, Lainnya{Menyebut Anda}}",
```

Comment:
We can't translate `other{...}` to `Lainnya{...}`. This `other` in this context is an important programmatic symbol that must always be `other`.

We can only translate inside of `{...}` :)